client/src/App.vue

```diff
@@ -1090,201 +1091,120 @@
+									<th>ID</th>
+									<th>Set</th>
+									<th>Players</th>
+									<th>Description</th>
+									<th>Join</th>
+								</tr>
+							</thead>
+							<tbody>
+								<tr v-for="s in publicSessions" :key="s.id">
+									<td :title="s.id" class="id">{{ s.id }}</td>
+									<td
+										v-tooltip="
+											s.cube
+												? 'Cube'
+												: s.sets.map((code) => setsInfos[code].fullName).join(', ')
+										"
+									>
+										<template v-if="s.cube">
+											<img src="./assets/img/cube.png" class="set-icon" />
+										</template>
+										<template v-else-if="s.sets.length === 1">
+											<img :src="setsInfos[s.sets[0]].icon" class="set-icon" />
+										</template>
+										<template v-else-if="s.sets.length === 0">All</template>
+										<template v-else>[{{ s.sets.length }}]</template>
+									</td>
+									<td>{{ s.players }} / {{ s.maxPlayers }}</td>
+									<td class="desc">{{ s.description }}</td>
+									<td>
+										<button v-if="s.id !== sessionID" @click="sessionID = s.id">Join</button>
+										<i
+											class="fas fa-check green"
+											v-tooltip="`You are in this session!`"
+											v-else
+										></i>
+									</td>
+								</tr>
+							</tbody>
+						</table>
 					</div>
-					<div class="container">
-						<div class="section-title">
-							<h2>News</h2>
-						</div>
-						<div class="welcome-section">
-							<div class="news">
-								<em>April 04, 2021</em>
-								<p><img src="img/sets/stx.svg" class="set-icon" style="--invertedness: 100%" /> Strixhaven: School of Mages support!</p>
-							</div>
-							<div class="news">
-								<em>March 28, 2021</em>
-								<p>
-									Introducing the <a href="https://www.mtgadraft.tk">MTGADraft.tk</a> domain.<br />
-									<i class="fas fa-exclamation-triangle yellow"></i> This is the exact same website but logs and preferences do not transfert over domains, you can still retrieve your game logs by accessing the <a href="https://mtgadraft.herokuapp.com">old URL</a>.  
-								</p>
-							</div>
-							<div class="news">
-								<em>March 06, 2021</em>
-								<p>
-									<img src="img/sets/tsr.svg" class="set-icon" style="--invertedness: 100%" />
-									Time Spiral Remastered (TSR) is now available! (see the "<i class="fas fa-ellipsis-h"></i> More sets..." option)<br />
-								</p>
-							</div>
-						</div>
+				</div>
+				<div class="container" style="grid-area: Tools;">
+					<div class="section-title">
+						<h2>Tools</h2>
 					</div>
-					<div class="container">
-						<div class="section-title">
-							<h2>Tools</h2>
-						</div>
-						<div class="welcome-section">
-							<ul>
-								<li>
-									<a @click="displayedModal = 'importdeck'">Card List Importer</a>
-								</li>
-							</ul>
-						</div>
+					<div class="welcome-section welcome-alt">
+						<ul>
+							<li>
+								<a @click="displayedModal = 'importdeck'">Card List Importer</a>
```

Comment:
Besides its name, I think the "export" icon fits here even better than the "import" icon because we are pulling the info from a file/list.
```suggestion
								<a @click="displayedModal = 'importdeck'"><i class="fas fa-file-export"></i> Card List Importer</a>
```